src/js/pages/nodes/breadcrumbs/NodeDetailBreadcrumb.js

```diff
@@ -0,0 +1,43 @@
+import BreadcrumbSegment from '../../../components/BreadcrumbSegment';
+import CompositeState from '../../../structs/CompositeState';
+
+class NodeDetailBreadCrumb extends BreadcrumbSegment {
+  constructor() {
+    super(...arguments);
+  }
+
+  componentDidMount() {
+    super.componentDidMount();
+
+    let hostname = this.getHostname();
+
+    if (hostname) {
+      this.setState({isLoadingCrumb: false});
+    }
```

Comment:
let's pull out the getting hostname + setting state into a method since we have the same exact code twice